plugins/defaultplugins/l2plugin/vppcalls/test/impl/channel_mocked_impl.go

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2017 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package impl
+
+import (
+	govppapi "git.fd.io/govpp.git/api"
+)
+
+//MockedChannel implements ChannelIntf for testing purposes
+type MockedChannel struct {
+	Channel govppapi.Channel
```

Comment:
This is currently unused.

Reply:
it is used in arp and also BD tests

Reply:
I meant only field `Channel` on line `23`, which is just set in tests, but never really used anywhere.